src/Validation/Validator.php

```diff
@@ -426,6 +426,9 @@ public function count()
      * then rules list for the field will be replaced with second argument and
      * third argument will be ignored.
      *
+     * The first argument can be an array of fields if you wish for the rule to
+     * apply to multiple fields.
```

Comment:
The rule would be applied to each field individually. This seems to imply that both fields would be arguments to one validation rule call.